### DISCUSSION TITLE
Weekly Update - 2022-06-07

### DISCUSSION THREAD

**Discussion by mjohnsonpint-stripe** [stripe/stripe-apps#643] [Announcements]
Hi Folks!

This is the FIRST community update for the Stripe Apps Beta.

Over the course of the beta, we will try to provide weekly or so updates on what is going on around Stripe Apps! Check back here for more information, or use the "watch" feature in GitHub.

## New Features
- Secrets API available in Stripe Client Libraries

## Tips and Tricks

Post Install actions give you more control over your user’s install experience!
If you haven’t dug in yet - before your next app review submission is a great time to make sure you have the right configuration for [post-install actions](https://stripe.com/docs/stripe-apps/post-install-actions).

After users install your app - this option supports linking to the app itself (default), but you can change it to app settings, 
```
    {
      "post_install_action": {
        "type": "settings"
      }
    }
```
or an external website
```
    {
      "post_install_action": {
        "type": "external",
        "url": "https://[YOUR-URL]"
      }
    }
```

## Community Buzz

We are thrilled to share these blog posts from the community about Stripe Apps!

- [Thin Platforms by Stratechery/Ben Thompson](https://stratechery.com/2022/thin-platforms/)
- [Creating Complex UIs Using the Stripe SDK by Dario Djurek at ThisDot](https://www.thisdot.co/blog/creating-complex-uis-using-stripe-sdk)
- [How to Log In to Third Party Services in Stripe Apps with OAuth PKCE by Tom Van Antwerp at ThisDot](https://www.thisdot.co/blog/how-to-login-in-to-third-party-services-in-stripe-apps-with-oauth-pkce)

## Pet of the Week
**Mickey**
There is nothing fierce about this little lion (“Shih Tzu”) who looked like a little mouse as a baby and hence was named Mickey. Mickey is now 4 years old. He rarely barks, usually cuddles, and loves to follow his family from room to room. Mickey is very possessive about his toys and will not share it with any other puppy
<img src="https://user-images.githubusercontent.com/92761515/172502004-2fae39ec-e045-4558-ad57-084ac747f2f5.jpg" width="300" />
<img src="https://user-images.githubusercontent.com/92761515/172502016-c191c4fe-dd9f-47bf-8130-2f5a6cc90a15.jpg" width="300">

.

**Comment by ahmed-adly-khalil** [comment]
what a cute dog :) the articles from partners is very nice and the whole stripe apps idea is very interesting, looking into it soon :)
